packages/sdk/src/evm/common/infra-data.ts

```diff
@@ -0,0 +1,42 @@
+import InfraData from "./InfraData.json";
+
+type InfraTxInfo = {
+  predictedAddress: string;
+  tx: string;
+  from: string;
+  deployData: string;
+};
+
+export const CloneFactory = {
+  name: "TWStatelessFactory" as const,
+  contractType: "cloneFactory" as const,
+  txInfo: InfraData["cloneFactory"] as InfraTxInfo,
+};
+
+export const NativeTokenWrapper = {
+  name: "WETH9" as const,
+  contractType: "nativeTokenWrapper" as const,
+  txInfo: InfraData["nativeTokenWrapper"] as InfraTxInfo,
+};
+
+export const Forwarder = {
+  name: "Forwarder" as const,
+  contractType: "forwarder" as const,
+  txInfo: InfraData["forwarder"] as InfraTxInfo,
+};
+
+export const EOAForwarder = {
+  name: "ForwarderEOAOnly" as const,
+  contractType: "eoaForwarder" as const,
```

Comment:
why do we need 'contractType' here? i think it makes things more complicated than needed here.

just have a map from contract name to InfraTxInfo